sandbox

```diff
@@ -430,13 +432,13 @@ sandbox () {
         rebuild_if_needed >> "$SANDBOX_LOG" 2>&1               & spinner
         echo "* docker-compose up -d"   >> "$SANDBOX_LOG"
         dc up -d                        >> "$SANDBOX_LOG" 2>&1 & spinner
-        sleep 1                                                & spinner
+        sleep 10                                               & spinner
```

Comment:
I guess this needs to be something in the 20-30 second range in order to succeed in your tests... that seems like too much just to get tests passing when end users are generally fine with 1 second.

I'm not sure the status is really needed though, how about leaving it at 1 second and removing the `status_helper` call just below these lines? That information might not be very useful so perhaps skip it to get tests running.

Reply:
This isn't acceptable, it means that in normal operation most of the startup time is an artificial sleep. You're welcome to create a new test outside of the production script, but I need to revert this change.